DOCUMENTATION.md

```diff
@@ -425,7 +425,6 @@ In order to batch push a list of data pools use the following command:
     "targetTeamDomain": "dev1",
     "dataPoolImportRequests": [
         {
-            "sourcePoolId": "850728cc-c679-4925-954a-87fb39abb12b",
```

Comment:
Why did we remove these? Will they not be needed in the file anymore?

Reply:
not needed